conf.js

```diff
@@ -1,11 +0,0 @@
-module.exports = {
-  username: process.env.HTTP_USERNAME,
```

Comment:
This will break things for the import map deployer running on https://microfrontends.app. Why did you delete it? A `conf.js` file works fine - if you undo your changes here I'll merge in the rest of the PR

Reply:
I thought it had to be a json file. Does it support both json and js config file?
Wasn't entirely clear. I'll fix it today.